composer.json

```diff
@@ -67,6 +66,7 @@
 		"wpackagist-plugin/pdf-embedder": "^4.6",
 		"wpackagist-plugin/simple-custom-css": "^4.0.3",
 		"wpackagist-plugin/spinupwp": "^1.1",
+		"wpackagist-plugin/woocommerce": "^4.9.2",
```

Comment:
why not using the last version of woocommerce?

Reply:
When using a more recent version, there was one of the existing WooCommerce test failing (`WooCommerceSubscriber::maybe_revert_uid_for_nonce_actions()` => `testShouldNotValidateNonce()`). I don't know why, so to be able to fix the problem at hand (composer v2 usage), I went with a version that was still working with this test.

We will want to have another look at the above method anyway, since it's likely no longer necessary (see #3929)

Reply:
aha, got your point.
Thanks.